pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/MailboxReceiveOperator.java

```diff
@@ -134,6 +138,7 @@ protected TransferableBlock getNextBlock() {
 
     int startingIdx = _serverIdx;
     int openMailboxCount = 0;
+    int eosMailboxCount = 0;
```

Comment:
nit: we can just use one int for openMailboxCount? When we receive EOS, we can do --openMailboxCount

Reply:
i think this code is easier to read as is - one variable for "the number of mailboxes opened" and one for "the number of EOS received". It isn't immediately clear why I should decrement the first when I get the second.